### PR TITLE
Lime 700 Take outstanding DL tests from the API repo

### PR DESCRIPTION
**Pull request by MathumithaVe**
Take outstanding DL tests from the API repo to Frontend repo

https://govukverify.atlassian.net/browse/LIME-700

 Proposed changes

Add the remaining front end test to Frontend repo

### CODE REVIEW

File: test/browser/features/DVLADrivingLicence.feature [hunk 1]
```diff
@@ -19,183 +19,198 @@ Feature: DVLA Driving licence CRI Error Validations
...
+    And I check the page Title Error: Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
     Examples:
       |DrivingLicenceSubject          |InvalidPostcode   |
       |DrivingLicenceSubjectHappyPeter|NW* ^%G           |
 
 ###### PostcodeWithNumericChar, PostcodeWithAlphaChar #####
-  @mock-api:dvla-invalidPostcode @validation-regression @build @staging
-  Scenario Outline: DVLA Driving Licence Postcode with numeric characters or alpha characters error validation
+  @mock-api:dl-success @validation-regression @build @staging
+  Scenario Outline: DVLA Driving Licence Postcode with numeric characters or Postcode with numeric characters error validation
```
Comment (chinmayi-gds) [inline test/browser/features/DVLADrivingLicence.feature]: Hi @MathumithaVe Could you please revert this line to what it was before i.e. Change this line 'Scenario Outline: DVLA Driving Licence Postcode with numeric characters or Postcode with numeric characters error validation' to 'Scenario Outline: DVLA Driving Licence Postcode with numeric characters or alpha characters error validation'

File: test/browser/features/DVA_Driving_Licence.feature [hunk 2]
```diff
@@ -15,17 +15,19 @@ Feature: DVA Driving licence CRI Error Validations
     When User clicks on continue
     Then I see the DVA licence number error in the summary as Your licence number should be 8 characters long
     And I can see the DVA licence number error in the field as Your licence number should be 8 characters long
+    And I check the page Title Error: Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
     Examples:
       |DVADrivingLicenceSubject       |InvalidLicenceNumber|
       |DrivingLicenceSubjectHappyBilly|5566778             |
 
-  @mock-api:dva-invalidDrivingLicenceNumber @validation-regression @build @staging
+  @mock-api:dva-invalidDrivingLicenceNumberWithSplChar
```
Comment (gds-jbentham) [inline test/browser/features/DVA_Driving_Licence.feature:23]: Why are we removing tags here?

File: test/browser/features/DVLADrivingLicence.feature [hunk 3]
```diff
@@ -278,6 +298,104 @@ Feature: DVLA Driving licence CRI Error Validations
...
+
+  @mock-api:dl-failed @validation-regression @build @staging
+  Scenario Outline:  DVLA Driving Licence number validation test - (VALID, female licenceNumber DOB Dec)
+    Given User enters DVLA data as a <DrivingLicenceSubject>
+    And User re-enters drivingLicenceNumber as <InvalidLicenceNumber>
+    And User re-enters day of birth as <InvalidDayOfBirth>
+    And User re-enters month of birth as <InvalidMonthOfBirth>
+    And User re-enters year of birth as <InvalidYearOfBirth>
+    When User clicks on continue
+    Then Proper error message is displayed as We could not find your details
```
Comment (gds-jbentham) [inline test/browser/features/DVLADrivingLicence.feature:348]: Is this correct should this not trigger the DoB and doc number validation message?

File: test/browser/features/DVA_Driving_Licence.feature (hunk 2 above)
Comment (MathumithaVe) [reply test/browser/features/DVA_Driving_Licence.feature]: Changed to the existing tag and committed the changes.

File: test/browser/features/DVLADrivingLicence.feature (hunk 3 above)
Comment (gds-jbentham) [reply test/browser/features/DVLADrivingLicence.feature:348]: Ignore this my validation logic was wrong

File: test/browser/features/DVA_Driving_Licence.feature [hunk 4]
```diff
@@ -226,6 +242,18 @@ Feature: DVA Driving licence CRI Error Validations
...
       |DVADrivingLicenceSubject       |InvalidValidToDay|InvalidValidToMonth|InvalidValidToYear|
       |DrivingLicenceSubjectHappyBilly|         10      |     01            |         2010     |
+
+  @mock-api:dva-ConsentError @validation-regression @build @staging
+  Scenario Outline:  DVA Driving Licence error validation when DVA consent checkbox is unselected
+    Given User enters DVA data as a <DVADrivingLicenceSubject>
+    And DVA consent checkbox is unselected
+    When User clicks on continue
+    Then I can see the DVA consent error summary as You must give your consent to continue
+    And I can see the DVA consent error on the checkbox as Error: You must give your consent to continue
```
Comment (gds-jbentham) [inline test/browser/features/DVA_Driving_Licence.feature:256]: Do we not also want to check the tab title here?

File: test/browser/features/DVLADrivingLicence.feature [hunk 5]
```diff
@@ -278,6 +298,104 @@ Feature: DVLA Driving licence CRI Error Validations
...
       |DrivingLicenceSubject          |InvalidValidToDay|InvalidValidToMonth|InvalidValidToYear|
       |DrivingLicenceSubjectHappyPeter|         10      |     01            |         2010     |
+
+  @mock-api:dl-success @validation-regression @build @staging
+  Scenario Outline:  DVLA Driving Licence error validation when DVLA consent checkbox is unselected
+    Given User enters DVLA data as a <DrivingLicenceSubject>
+    And DVLA consent checkbox is unselected
+    When User clicks on continue
+    And I can see the DVLA consent error on the checkbox as Error: You must give your consent to continue
+    Then I can see the DVLA consent error summary as You must give your consent to continue
```
Comment (gds-jbentham) [inline test/browser/features/DVLADrivingLicence.feature:312]: Same here as DVA

File: test/browser/features/DVA_Driving_Licence.feature (hunk 4 above)
Comment (MathumithaVe) [reply test/browser/features/DVA_Driving_Licence.feature:256]: Tab title validation added.

**Review by gds-jbentham — APPROVED** [review]

**Pull request merged**